src/3rdparty/kissfft/test/kissfft-testsuite.sh

```
@@ -0,0 +1,275 @@
+	# Run selected action
+
+	echo "$_STATUS_LINE"
+
+	case "$_ACTION" in
+		"test")
+			_MAKE_OPTS="$_MAKE_OPTS PREFIX=$_INSTALL_DIR"
+			_CMAKE_OPTS="$_CMAKE_OPTS -DCMAKE_INSTALL_PREFIX=$_INSTALL_DIR"
+
+			case "$_BUILD_TYPE" in
+				"make")
+					make $_MAKE_OPTS all 1>>"$_LOG_FILE" 2>&1 &&
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)